### ISSUE TITLE
the problem of loss = nan

### ISSUE THREAD

**Issue by luoshiyong** [MIC-DKFZ/nnUNet#941]
hi! I want to use nnunet to solve the lits17 dataset ,but the train loss and val loss is always nan,i don't know how to fixd it. Can you give me some suggestions?
The following is my process using nnunet
1. git clone https://github.com/MIC-DKFZ/nnUNet.git
cd nnUNet
pip install -e .
2. Convert lits data format: python Task029_LiverTumorSegmentationChallenge.py
3. Preprocess data: nnUNet_plan_and_preprocess -t 029
4. Train 2d network: nnUNet_train 2d nnUNetTrainerV2 Task029_Liver 0 --npz

**Comment by FabianIsensee** [comment]
Hi,
that is strange. I cannot reproduce this problem. Can you please try Task03 from the medical segmentation decathlon? It's the same dataset as LiTS, just from a different source and with some corrected spacings
Best,
Fabian

**Comment by Kobalt93** [comment]
Have you tried the suggested solutions? Could you please give us feedback?
